pkg/exchange/bybit/convert.go

```diff
@@ -0,0 +1,53 @@
+package bybit
+
+import (
+	"math"
+
+	"github.com/c9s/bbgo/pkg/exchange/bybit/bybitapi"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+func toGlobalMarket(m bybitapi.Instrument) types.Market {
+	// sample:
+	//Symbol: BTCUSDT
+	//BaseCoin: BTC
+	//QuoteCoin: USDT
+	//Innovation: 0
+	//Status: Trading
+	//MarginTrading: both
+	//
+	//LotSizeFilter:
+	//{
+	//    BasePrecision: 0.000001
+	//    QuotePrecision: 0.00000001
+	//    MinOrderQty: 0.000048
+	//    MaxOrderQty: 71.73956243
+	//    MinOrderAmt: 1
+	//    MaxOrderAmt: 2000000
+	//}
+	//
+	//PriceFilter:
+	//{
+	//    TickSize: 0.01
+	//}
```

Comment:
Could you add a unit test for this converting function to make sure parsing is performed as expected?

Reply:
sure!

Reply:
Yeah that would be better